great_expectations/datasource/fluent/file_path_data_asset.py

```diff
@@ -11,6 +11,7 @@
     ClassVar,
     Dict,
     List,
+    Mapping,
```

Comment:
@Kilo59 I see `Mapping` defined here, but not used in the present module; am I missing something here?  Thanks.

Reply:
It is used on line `65`